app/services/rapid_ready_for_decision/views/shared/medications.erb

```diff
@@ -35,25 +40,19 @@
         <%= medication['dosageInstructions'].join('; ') %><br />
       <% end %>
 
-      <% if medication['authoredOn'].present? %>
-        Prescribed on: <%= medication['authoredOn'][0, 10].to_date.strftime('%m/%d/%Y') %>
-      <% end %>
+      <%# TODO: Add test for additional fields and make sure to include additional medications %>
```

Comment:
Actually, given what we learned about nil values, we should include the "present?" check...